bin/check_samplesheet.py

```diff
@@ -136,11 +142,6 @@ def check_samplesheet(file_in, file_out):
         with open(file_out, "w") as fout:
             fout.write(",".join(["sample", "single_end", "rg_id", "rg_lb", "rg_pl", "fastq_1", "fastq_2"]) + "\n")
             for sample in sorted(sample_mapping_dict.keys()):
-
-                ## Check that multiple runs of the same sample are of the same datatype
-                if not all(x[0] == sample_mapping_dict[sample][0][0] for x in sample_mapping_dict[sample]):
-                    print_error("Multiple runs of a sample must be of the same datatype!", "Sample", "{}".format(sample))
```

Comment:
You expect the same sample to be sequenced as both paired-end and single-end?

Reply:
This would be unusual but it certainly could happen. The tool doesn't have this requirement, so I'd rather not enforce it in the pipeline, if possible.